runtime/zeitgeist/src/parachain_params.rs

```diff
@@ -107,71 +89,20 @@ parameter_types! {
     pub const RewardPaymentDelay: u32 = 2;
 
     // XCM
+    /// Base weight for XCM execution
+    pub const BaseXcmWeight: Weight = 200_000_000;
+    /// The maximum number of distinct assets allowed to be transferred in a
+    /// single helper extrinsic.
+    pub const MaxAssetsForTransfer: usize = 2;
+    /// Max instructions per XCM
     pub const MaxInstructions: u32 = 100;
+    // Relative self location
+    pub SelfLocation: MultiLocation = MultiLocation::new(1, X1(Parachain(ParachainInfo::get().into())));
```

Comment:
What do you get with the parachain info getter here?

From above: Do you query the parachain identifier here with `parachain_info` (Why not `::parachain_id()` then)? What happens, when the parachain identifier changes (like it will happen for the main-net from `2101` to another id). Because the identifier is part of the Genesis configuration, we have to keep all usages of parachain_info in mind, right?

Reply:
Both functions are identical, [`get` is achieved by a trait implementation](https://github.com/paritytech/cumulus/blob/3a624fd8ea8f11df3594ccde7dd6d38b8c3afee5/parachains/pallets/parachain-info/src/lib.rs#L71-L75).
If `parachain_info::parachain_id()` changes, the chain would not be able to recognize incoming ZTG (from other chains) as it's native currency and instead would regard it as an unknown token, unless registered in `AssetRegistry`.
There is no way to directly change the parachain id via a dispatchable, except using `system.setStorage` which always indicates the danger of unforeseen implications.

Reply:
Could we be consistent with the way of getting the parachain id? Just a nitpick. I would be in favour of using `::parachain_id()`.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/661/commits/54d388935551b3afcb43e30c27013c387e0c2c4e